### PR TITLE
Partial JSON Purge

### PR DESCRIPTION
**Pull request by CjBuresch**
+A good part of JSON is now out of handler.
+AcceptedRPC is cleaner now.
+Chnaged TableResult, IndexResult, and ValueResult from protected to public
+RPCValidator is kind of dead code at this point.
+ServerUnittests is done now with RPC objects now and has been confirmed to have working PING
+Lots of fixes?

### CODE REVIEW

File: src/blackdoor/cqbe/rpc/RPCValidator.java [hunk 1]
```diff
@@ -24,57 +18,46 @@
...
+			if (isValid(request)) {
+				RPCHandler handler = new RPCHandler(io);
+				response = handler.handle(request);
+			} else
+				response = buildError(request);
+		} catch (RPCException e) {
+			// problems unpacking
+			response = new ErrorRpcResponse(request, e.getRPCError());
+		} catch (IOException e) {
+			// TODO What do? shutdown had issues.....
```
Comment (euphzji) [inline src/blackdoor/cqbe/rpc/RPCValidator.java]: Now that shutdown is fixed does this need something?

File: src/blackdoor/cqbe/test/CLITester.java [hunk 2]
```diff
@@ -19,17 +19,22 @@ public static void main(String[] args) throws Exception, IOException {
...
-		//String[] args2 = {"insert","file.txt","-b", address.getHostAddress() + ":1776"};
-		//String[] args2 = {"join",address.getHostAddress() + ":1776","-s","savefile.txt"};
-		//String[] args2 = {"join","-a"};
-		String[] args2 = {"join", address.getHostAddress() + ":1776","-p","1779","-dm"};
-		//String[] args2 = {"join", address.getHostAddress() + ":1776","-p","1780"};
-		//String[] args2 = {"join", address.getHostAddress() + ":1776","-p","1781"};
-		//String[] args2 = {"join", address.getHostAddress() + ":1776","-p","1782"};
-		//String[] args2 = {"join", address.getHostAddress() + ":1776","-p","1783"};
-		//String[] args2 = {"shutdown", "-p", "1776"};
+		DBP.DEMO= true;
```
Comment (euphzji) [inline src/blackdoor/cqbe/test/CLITester.java]: This file doesn't have DBP imported. Is it tested?

**Comment by euphzji** [comment]
Apart from the DBP import this looks good!

File: src/blackdoor/cqbe/rpc/RPCValidator.java (hunk 1 above)
Comment (CjBuresch) [inline src/blackdoor/cqbe/rpc/RPCValidator.java]: eh. validator has been merged over to handler, so this is dead code

File: src/blackdoor/cqbe/test/CLITester.java (hunk 2 above)
Comment (CjBuresch) [inline src/blackdoor/cqbe/test/CLITester.java]: this is just a class that peeps use to run instances of a node locally and doesnt really "test" things

**Pull request closed without merging**